src/Sly/NotificationPusher/Adapter/Gcm.php

```diff
@@ -138,11 +138,11 @@ public function getServiceMessageFromOrigin(array $tokens, BaseOptionedModel $me
     public function getDefinedParameters()
     {
         return [
-            'collapse_key',
```

Comment:
@theluk, now I have examined the code and I see, that you made parameter names all in camel case everywhere. Now they are consistent and it's ok. :+1: